packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -639,7 +988,6 @@ class ListWheelViewport extends MultiChildRenderObjectWidget {
       itemExtent: itemExtent,
       clipToSize: clipToSize,
       renderChildrenOutsideViewport: renderChildrenOutsideViewport,
-      offset: offset,
     );
   }
```

Comment:
everything that's here also need to be in the updateRenderObject below

Reply:
They are, I'm just reordering arguments here.